pkgs/by-name/un/unyaffs/package.nix

```
@@ -0,0 +1,40 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenv,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "unyaffs";
+  version = "0.9";
+
+  src = fetchFromGitHub {
+    owner = "whataday";
+    repo = "unyaffs";
+    rev = "${finalAttrs.version}";
+    hash = "sha256-FcuPaxq74gVJ6UlOhFPSMtuwUAJVV/sAxAQojhmVXCs=";
+  };
+
+  preBuild = ''
+    sed -i '54i #if defined(__linux__)' unyaffs.c
+    sed -i '55i #include <sys/sysmacros.h>' unyaffs.c
+    sed -i '56i #endif' unyaffs.c
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    install -D unyaffs -t $out/bin
```

```suggestion
    install -Dm755 unyaffs -t $out/bin
```

To prevent the build system from not marking us the executable bit...